src/main/java/com/glitchcog/fontificator/emoji/loader/twitch/TwitchBadgesNew.java

```diff
@@ -0,0 +1,77 @@
+
+        package com.glitchcog.fontificator.emoji.loader.twitch;
+
+
+public class TwitchBadgesNew {
```

Comment:
This is a rather silly name for this class, but it could always be renamed, or eventually just replace the other one.